src/components/CategoryIndicator/CategoryIndicator.js

```diff
@@ -38,6 +38,7 @@ const CategoryIndicator = (props) => {
           className={style.card}
           indicator={props.category.heroIndicatorId}
           color={categoryColor}
+          colorName={props.category.colorName}
```

Comment:
Why are both color and colorName required ?

Reply:
Yeah it's a bit of a mess. The `color` is an actual hex code. It needs to be passed in explicitly because it's different for every sub-category, so the parent component must calculate it.

But also, we need to just take the 900 variant of whatever the current category color is (`colorName`) for the `<IndicatorTypeMark>`, and we don't want to calculate that multiple times and pass it down as a HEX code.